setup.py

```diff
@@ -35,8 +15,7 @@
     url='https://github.com/JulienPalard/turtle-backup',
     entry_points={
         'console_scripts': [
-            'turtl-backup=turtl_backup.turtl_backup:main',
-            'turtl-backup-gui=turtl_backup.gui:main'
```

Comment:
Why removing the `console_scripts` `turtl-backup-gui`?

Reply:
yes, you're right, fix in #b52a047f03456d55a91c718c0091b94ac302d706